### PR TITLE
Setup for users

### LINKED ISSUE #50: documentation
- requirements, installation, set up, usage, testing

### PR DESCRIPTION
**Pull request by stefpiatek**
- Added documentation; closes #50 
- Made cnv-patissier.py runner user friendly
- Slowly been working on it for a while, sorry that it's so huge!

Feel free to ignore the sample-sheet files and bed files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,189 @@
 # CNV-patissier
 
-Orchestrates your CNV bakeoff, part of research project for Clinical Bioinformatics Scientist Training Programme. 
+Orchestrates your Copy Number Variant (CNV) bakeoff, part of research project for Clinical Bioinformatics Scientist Training Programme. 
+
+## Overview
+
+This project is aimed to collect data on the ability of CNV callers (targeted-capture Next Generation Sequencing) to detect clinically relevant CNVs. All samples should have a gold-standard known-CNV status for the gene in question. This project is aimed at bioinformaticians working genetics laboratories but please let me know via the issues if there are any problems in installation or running, regardless.
```
Comment (mlgosh) [inline README.md]: I think you're missing an "in".

Should this be:
"aimed at bioinformaticians working in genetics laboratories"

File: README.md [hunk 2]
```diff
@@ -1,3 +1,189 @@
...
+## Setup and usage
+
+### Overview of steps taken by CNV-patissier
+
+- CNV-patissier is run per capture and scans the gene sample sheets and runs all CNV callers on every sample per gene. 
+    - The output from each caller is created in `cnv-patissier/output/<capture>/<date-time-of-run>/<cnv-caller>/`
+    - The known CNV status and called CNVs are saved in a sqlite database in `cnv-patissier/output/` 
+    - A successful run settings file is written to `cnv-patissier/successful-run-settings/<capture>/<cnv-caller>/<gene>.toml`
+    - Logs are written to `cnv-patissier/logs/`
+- Each caller checks if there as been a successful run for that gene, if there has an no settings have changed (i.e. sample paths) then it moves onto the next.
```
Comment (mlgosh) [inline README.md]: Just a couple of typo's on this line:

Should this be:
"Each caller checks if there has been a successful run for that gene, if there has and no settings have changed (i.e. sample paths) then it moves onto the next. "

Comment (stefpiatek) [reply README.md]: thanks maria! corrected.

File: README.md (hunk 1 above)
Comment (stefpiatek) [reply README.md]: :+1:

**Pull request merged**